plugins/osdn/multitenant/plugins.go

```diff
@@ -69,19 +68,5 @@ func (plugin *MultitenantPlugin) TearDownPod(namespace string, name string, id k
 }
 
 func (plugin *MultitenantPlugin) Status(namespace string, name string, id kubeletTypes.DockerID) (*knetwork.PodNetworkStatus, error) {
-	vnid, err := plugin.getVnid(namespace)
-	if err != nil {
-		return nil, err
-	}
-	out, err := utilexec.New().Command(plugin.getExecutable(), statusCmd, namespace, name, string(id), strconv.FormatUint(uint64(vnid), 10)).CombinedOutput()
-	glog.V(5).Infof("PodNetworkStatus 'multitenant' network plugin output: %s, %v", string(out), err)
-	if err != nil {
-		return nil, err
-	}
-	var podNetworkStatus knetwork.PodNetworkStatus
-	err = json.Unmarshal([]byte(out), &podNetworkStatus)
-	if err != nil {
-		return nil, err
-	}
-	return &podNetworkStatus, nil
+	return &knetwork.PodNetworkStatus{}, nil
```

Comment:
That needs to be "return nil, nil" doesn't it? (look at how it gets used in k8s/pkg/kubelet/dockertools/manager.go)